introduction.md

```diff
@@ -1,3 +1,7 @@
 This is jp-inet's introduction file. My favorite track at Code School is
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?